src/main/java/hoodini/Hoodini.java

```diff
@@ -0,0 +1,48 @@
+package hoodini;
+
+/**
+ * Handles the main configurations
+ * of the Chatbot
+ */
+
+public class Hoodini {
+    private Storage store;
+    private Parser parser;
+    private Ui ui;
+
+    /**
+     * Starts the Hoodini chatbot, takes no parameters.
+     */
+
+    public Hoodini() {
+        this.ui = new Ui();
+        this.store = new Storage(ui);
+        start();
+
+
+
+
+    }
```

Comment:
I've noticed here as well as in other parts of your code that there are a lot of extra lines. Maybe you could tidy up your code by keeping the end braces just one line below the last line of code in that block?